openedx/core/djangoapps/catalog/utils.py

```diff
@@ -163,6 +164,29 @@ def get_programs_by_type(site, program_type):
     return get_programs_by_uuids(uuids)
 
 
+def get_programs_by_type_slug(site, program_type_slug):
+    """
+    Keyword Arguments:
+        site (Site): The corresponding Site object to fetch programs for.
+        program_type_slug (string): The type slug that matching programs must have.
+
+    Returns:
+        A list of programs for the given site with the given type slug.
+
+    Slugs are a consistent identifier whereas type (used in `get_programs_by_type`
```

Comment:
nit: Missing the closing parenthesis in docstring?

Reply:
Fixed!